modules/admin/src/test/java/org/motechproject/admin/it/AdminBundleIT.java

```diff
@@ -114,6 +127,30 @@ public void testMessageController() throws IOException, InterruptedException {
         assertTrue("No messages listed", json.size() > 0);
     }
 
+    @Test
+    public void testUploadBundleFromRepository() throws IOException, InterruptedException {
+        Bundle[] bundlesBeforeUpload, bundlesAfterUpload;
```

Comment:
1. No need to define those arrays beforehand, just do it together with the initialization.
2. Since we do not need the bundles themselves and are only interested in the number of bundles, can we just store ints?
3. We try to follow the one field/variable per line rule, although it's not strictly enforced in tests.